filters/include/pcl/filters/impl/morphological_filter.hpp

```diff
@@ -52,155 +52,157 @@
 #include <pcl/filters/morphological_filter.h>
 #include <pcl/octree/octree_search.h>
 
-///////////////////////////////////////////////////////////////////////////////////////////
-template <typename PointT> void
-pcl::applyMorphologicalOperator (const typename pcl::PointCloud<PointT>::ConstPtr &cloud_in,
-                                 float resolution, const int morphological_operator,
-                                 pcl::PointCloud<PointT> &cloud_out)
+namespace pcl
 {
-  if (cloud_in->empty ())
-    return;
+  template <typename PointT> void
```

Comment:
You've indented this entire block, which is against the norm as well as the coding standard.

Reply:
Maybe clang.format can have `NamespaceIndentation: None` added to it? I'm pretty sure there's no way to format newlines at the end of files, I searched the other day before doing all those newlines additions which I added with the bad force push.

Reply:
I thought it had `NamespaceIndentation: None`. No issue, #3780 is here to rescue. filters is not in the whitelist yet though.